app/javascript/packages/document-capture/components/documents-step.jsx

```diff
@@ -47,16 +48,26 @@ function DocumentsStep({
    *
    * @type {DocumentSide[]}
    */
-  const documentSides = selfieCaptureEnabled ? ['front', 'back', 'selfie'] : ['front', 'back'];
+  const documentSides = ['front', 'back'];
+  const selfieSide = 'selfie';
 
+  const pageHeaderText = selfieCaptureEnabled
+    ? t('doc_auth.headings.document_capture_with_selfie')
+    : t('doc_auth.headings.document_capture');
+
+  const idTipListTitle = t('doc_auth.tips.document_capture_selfie_id_header_text');
   return (
     <>
       {flowPath === 'hybrid' && <HybridDocCaptureWarning className="margin-bottom-4" />}
-      <PageHeading>{t('doc_auth.headings.document_capture')}</PageHeading>
-      <p>{t('doc_auth.info.document_capture_intro_acknowledgment')}</p>
+      <PageHeading>{pageHeaderText}</PageHeading>
+      <h2>
+        {selfieCaptureEnabled
+          ? `1. ${t('doc_auth.headings.document_capture_subheader_id')}`
```

Comment:
I think I see why you're putting the `1.` outside the translation string here, but it feels kind of odd to have it hardcoded. Also, if we ever add a language that doesn't use arabic numerals it might be better to have this in the translation string?

Reply:
@charleyf , that's what i did originally then find out for the current languages are the same on this. And if not all, at least most of the education system in the word have math classes that use 1, 2, 3. Like in simplified Chinese we have “一 二 三 四 五 六 七 八 九“ for "1 2 3 4 5 6 7 8 9", but in reality almost all places use 1, 2, 3 etc in daily life.

We can also argue we have date and calendars, do we want to support something like this
<img src="https://www.sakuramani.com/wp-content/uploads/JapaneseCalendarJan15.jpg" alt="Japanese Calendar" />

Yes, it's very nice to have, but it requires, another two entries in i18files for each language, not convinced it worths the effort for now.

Reply:
Ah, I missed that it's basically appending the `1.` on the front. Agreed, makes sense to leave it like this.